packages/button/src/components/ToggleButton.tsx

```diff
@@ -15,16 +15,10 @@ class ToggleButton extends React.Component<
   ToggleButtonProps,
   ToggleButtonStates
 > {
-  static defaultProps = {
-    active: false
+  state = {
+    active: this.props.active || false
   };
 
-  static getDerivedStateFromProps(props: ToggleButtonProps) {
-    return {
-      active: props.active
```

Comment:
since `getDerivedStateFromProps` always run right before `render`, the `props.active` will always overwrite the current state